bool add_buffer # adds the frames to the frame buffer -> max 10 frames can be stored
---
int64 imgID